### ISSUE TITLE
Run through LDN Test Suite

### ISSUE THREAD

**Issue by csarven** [CenterForDigitalHumanities/inbox#16]
It'd be great to see this pass the tests at https://linkedresearch.org/ldn/tests/ so that we can see it at https://linkedresearch.org/ldn/tests/summary :)

**Comment by cubap** [comment]
I agree! Thank you for sharing and let's get some action back in this place.
